### PR TITLE
Fix documentation about readmodels

### PR DESCRIPTION
**Pull request by Nek-**
Fix **a lot** of headache on 2 points:
1. add the notice about the dash interpretation of eventstore
2. Do not use `$resolvedEvent->originalEvent()` (which is wrong) but `$resolvedEvent->event()`

I also added how to handle errors, just because it's a must-have and not that obvious. (Actually I was like "excuse me wtf? amp is precisely designed to be able to throw exception >.>'", then I figure out it was to follow the .NET API, which is completely legit)

### CODE REVIEW

File: event-store-client/readmodels.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ c) Write a script that subscribes to it.
 * * *
 
 Your event streams should be organized as one stream per aggregate. If you have 5 users in your system and the aggregate ids are simply `1`, `2`, `3`, `4`, `5`, then you may have 5 streams with these names:
-`user-1`, `user-2`, `user-3`, `user-4`, `user-5`.
+`user-1`, `user-2`, `user-3`, `user-4`, `user-5`. Your stream name (before id) **cannot contain dashes** because eventstore use them with this patern `aggregateCategory-{id}` (id can contain dashes).
```
Comment (prolic) [inline event-store-client/readmodels.md]: 1) typo: "pattern" not "patern".
2) using "dash" is only the default, it's something you can configure, f.e. you can use "." instead, if you prefer that.

File: event-store-client/readmodels.md [hunk 2]
```diff
@@ -108,9 +117,9 @@ class UserEventAppeared
         ?int $retryCount = null
     ): Promise
     {
-        switch ($resolvedEvent->originalEvent()->eventType()) {
+        switch ($resolvedEvent->event()->eventType()) {
             case 'user-registered':
-                $data = $resolvedEvent->originalEvent()->data();
+                $data = $resolvedEvent->event()->data();
```
Comment (prolic) [inline event-store-client/readmodels.md:122]: fyi: the original event is the link-event, because that's what you subscribed to.

**Review by prolic — CHANGES_REQUESTED** [review]

File: event-store-client/readmodels.md (hunk 1 above)
Comment (prolic) [reply event-store-client/readmodels.md]: for configuration see here: https://eventstore.com/docs/projections/system-projections/index.html?tabs=tabid-5

File: event-store-client/readmodels.md (hunk 2 above)
Comment (Nek-) [reply event-store-client/readmodels.md:122]: I finally got it but wow, was hard to understand. (since there's almost no doc about it)

**Review by prolic — APPROVED** [review]

**Pull request merged**